utils/build/docker/ruby/graphql23/config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```

Comment:
same as above
If this file is not necessary for the Rails app to work, can we remove it ?